apps/frontend/src/components/Surface.tsx

```diff
@@ -13,8 +13,8 @@ export const Surface = forwardRef<HTMLDivElement, SurfaceProps>(
     ref
   ) => {
     const surfaceClass = cn(
-      className,
       "bg-white rounded-xl dark:bg-neutral-900",
```

Comment:
```suggestion
      "rounded-xl",
```

removed background colors so we can use the external classname colors.